capstone/src/main/webapp/youtube_analyzer.html

```diff
@@ -68,6 +69,7 @@ <h1>Youtube Analysis</h1>
         <div id="available-attributes"></div>
         <button onclick="callYoutube()" class="btn btn-primary">Submit</button>
         <button onclick="getKeywordSearchResults()" class="btn btn-primary" id="keywordSearch">Keyword Search</button>
+        <button onclick="beginDownload()" class="btn btn-primary" id="keywordSearch">Download Output</button>
```

Comment:
We might want to have different `id`s for the two buttons above, as it might cause conflicts later.

Reply:
addressed in "address Daniel's and Chris's nits"